nautobot/dcim/templates/dcim/location.html

```diff
@@ -173,7 +173,7 @@
         <div class="panel-heading">
             <strong>Stats</strong>
         </div>
-        <div class="row panel-body">
+        <div class="panel-body">
```

Comment:
This was causing the panel to extend outwards of the panel boundary it is now fixed.
![Screen Shot 2023-11-20 at 12 33 14 PM](https://github.com/nautobot/nautobot/assets/46973263/beae0aac-42fa-4e50-80ae-33c16d7444c6)

Reply:
![Screen Shot 2023-11-20 at 12 34 57 PM](https://github.com/nautobot/nautobot/assets/46973263/d06b63fc-14f3-42a5-a886-d045cf109b30)